src/Template/Bake/Element/Controller/add.ctp

```diff
@@ -18,7 +18,7 @@ $compact = ["'" . $singularName . "'"];
     /**
      * Add method
      *
-     * @return void Redirects on successful add, renders view otherwise.
+     * @return \Cake\Network\Response|void Redirects on successful add, renders view otherwise.
```

Comment:
It's all semantics :) Semantically one would also need to always add return null explicitly to make the IDE happy once you start returning sth. But we at this point don't care about that too much.
I wonder if actions should be more like Symfony ones and always return an array otherwise with the view vars :) instead of setting them. But thats off topic now HEHE